script.js

```diff
@@ -15,12 +20,9 @@ function randInRange(min, max) {
 	var range = max-min;
 	return Math.floor(Math.random()*range)+min;
 }
-
 // write a function "setStyles" that accepts an element and sets all numerical CSS properties to random values within logically derived ranges.
 function setStyles(element) {
-	var windowWidth = window.innerWidth,
-		windowHeight = window.innerHeight,
-		divWidth = randInRange(1,600),
+	var divWidth = randInRange(1,600),
 		divHeight = randInRange(1,500),
 		leftMax = windowWidth - divWidth,
 		topMax = windowHeight - divHeight;
```

Comment:
That issue with the div height/width also mean that leftMax and topMax could be negative.